.github/workflows/pipeline.yaml

```diff
@@ -226,6 +226,10 @@ jobs:
           sudo chmod +x /usr/bin/hadolint
           hadolint --version
 
+      - name: Run lint
```

Comment:
Lint (`make lint`) requires to be done on the sources, not on the fly. That's why we are testing the quality with `make test`.